scripts/backout_data_collection.py

```diff
@@ -0,0 +1,219 @@
+import sys
+import logging
+import json
+import os
+from tqdm import tqdm
+
+sys.path.append('../bugbug')
+from bugbug import repository, db, bugzilla
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+def download_commits() -> None:
+    """
+    Download the database of all commits.
+
+    Args:
+        None
+
+    Returns:
+        None
+    """
+
+    logger.info("Downloading commits database...")
+    assert db.download(repository.COMMITS_DB, support_files_too=True)
+
+def download_bugs() -> None:
+    """
+    Download the database of all bugs.
+
+    Args:
+        None
+
+    Returns:
+        None
+    """
+
+    logger.info("Downloading bugs database...")
+    assert db.download(bugzilla.BUGS_DB)
+
+def is_bug_fixed(bug_id: int) -> bool:
+    """
+    Check if a bug (given its ID) is resolved.
+
+    Args:
+        bug_id (int): the ID of the bug from Bugzilla
+
+    Returns:
+        bool: True if the bug is RESOLVED, False otherwise
+    """
+
+    # get bug details
+    bug_details = bugzilla.get([bug_id])
```

Comment:
The `bugzilla.get()` method is a bit expensive since it performs an API request on each call. We could utilize the bugs db for that.